src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -38,6 +38,15 @@
 #[AsCommand(name: 'debug:config', description: 'Dump the current configuration for an extension')]
 class ConfigDebugCommand extends AbstractConfigCommand
 {
+    public function __construct(string $name = null)
+    {
+        if (!class_exists(Yaml::class)) {
+            throw new \RuntimeException('The ConfigDebugCommand class requires the "Yaml" component. Install "symfony/yaml" to use it.');
```

Comment:
"Install "symfony/yaml" to use it."

In other code parts we use something like:

"Try composer require symfony/yaml...."